.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
sorry I hadn't seen that, I don't think we should add editor dotfiles to the project

Reply:
Oh, my mistake... should I push a new commit to the brach with this file deleted?

Reply:
yes, I wanted to do it as a suggestion but it does not seem possible

Reply:
nevermind it works, I committed it